tools/protobuf-relocated/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+plugins {
+  `java-library`
+  `maven-publish`
+  signing
+  id("org.projectnessie.buildsupport.reflectionconfig")
+  `nessie-conventions`
+}
+
+apply<NessieShadowJarPlugin>()
+
+extra["maven.name"] = "Nessie - Relocated Protobuf ${libs.protobuf.java.get().version}"
```

Comment:
nit: Shall we state the full protobuf's Maven coordinates here?

Reply:
hm - it's just a name though